build.sbt

```diff
@@ -198,10 +203,47 @@ lazy val sourcing = project
     ),
     Test / fork := true
   )
+lazy val rdf = project
+  .in(file("rdf"))
+  .settings(shared, compilation, coverage, release)
+  .settings(
+    name            := "rdf",
+    moduleName      := "rdf"
+  )
+  .settings(
+    libraryDependencies ++= Seq(
```

Comment:
there are a lot of dependencies here are don't seem to be justified, like: kryo, akkaPersistence etc. please remove them.